docs/content/how-tos/rules/FL0077.md

```diff
@@ -0,0 +1,28 @@
+---
+title: FL0077
+category: how-to
+hide_menu: true
+---
+
+# UsedUnderscorePrefixedElements (FL0077)
```

Comment:
@realmarv other rules' documentation states the version where the rule is introduced, please do same here

Reply:
Done